jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1012,37 +1000,29 @@ code_blockt java_bytecode_convert_methodt::convert_instructions(
     // a new maximal key
     assert(a_entry.first==--address_map.end());
 
-    if(i_it->statement!="goto" &&
-       i_it->statement!="return" &&
-       !(i_it->statement==patternt("?return")) &&
-       i_it->statement!="athrow" &&
-       i_it->statement!="jsr" &&
-       i_it->statement!="jsr_w" &&
-       i_it->statement!="ret")
+    const std::string statement = bytecode_info[i_it->bytecode].mnemonic;
```

Comment:
Const ref?

Reply:
No, this is turning a `const char *` into a string, for comparison.
Won't fix now, since this will be replaced by bytecode comparisons.